components/verifalia/actions/delete-job/delete-job.mjs

```diff
@@ -0,0 +1,28 @@
+import verifalia from "../../verifalia.app.mjs";
+
+export default {
+  name: "Delete Email Verification Job",
+  description: "Delete a previously submitted email verification job. " +
+        "[See the docs](https://verifalia.com/developers#email-validations-deleting) for more information",
+  key: "verifalia-delete-job",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    verifalia,
+    id: {
+      type: "string",
+      label: "Job ID",
+      description: "The ID of the email verification job to delete, for example `e0dda878-3801-43f3-ac20-935e27dbe5a7`",
+    },
+  },
+
+  async run() {
+    const verifaliaClient = this.verifalia.buildVerifaliaRestClient();
+
+    await this.verifalia.wrapVerifaliaApiInvocation(async () => {
```

Comment:
```suggestion
    this.verifalia.wrapVerifaliaApiInvocation(async () => {
```

Reply:
@lcaresia This would just make the code more error prone, in my opinion - think what would happen if that code block would be later enclosed in a `try/catch`. For the sake of clarity, I prefer to leave this as is, thanks.